### DISCUSSION TITLE
Admin Setup

### DISCUSSION THREAD

**Discussion by STARMEET** [EvanMulawski/BeamMP.EnvSync#21] [Q&A] [answered]
Hey,
I firstly want to thanks you for the mod, secondly, I'm habing a problem with the admin Setup, i don't seem to make it work xD
LG
![image](https://user-images.githubusercontent.com/65718240/219948962-c53e0d44-f22d-4d6a-a14e-dfc6c6187453.png)

**Comment by EvanMulawski** [comment, marked as answer]
Hi @STARMEET, please see https://github.com/EvanMulawski/BeamMP-EnvSync/wiki/Installation#admins and let me know if you're still having trouble.

**Comment by STARMEET** [comment]
Hey, thx for the fast answer, I tryed by following the wiki, changing the 12345 by my ID but doesn't work, Maybe I'm doing something wrong. 
Ok I'm stupid, like REALLY Stupid, i didn't saw that you put the `option.admins.beammp_XXXX = "X"` in commentary.
I'm stupid so sorry.
